nilrt_snac/_configs/_firewall_config.py

```diff
@@ -0,0 +1,101 @@
+            run(["firewall-offline-cmd", "--zone=public", "--remove-forward"])
+
+            run(["firewall-offline-cmd",
+                 "--new-policy=work-in"])
+            run(["firewall-offline-cmd",
+                 "--policy=work-in",
+                 "--add-ingress-zone=work"])
+            run(["firewall-offline-cmd",
+                 "--policy=work-in",
+                 "--add-egress-zone=HOST"])
+            run(["firewall-offline-cmd",
+                 "--policy=work-in",
+                 "--add-service=ssh",
+                 "--add-service=mdns"])
+            run(["firewall-offline-cmd",
+                 "--policy=work-in",
+                 "--set-target=REJECT"])
+
+            run(["firewall-offline-cmd",
+                 "--new-policy=work-out"])
+            run(["firewall-offline-cmd",
+                 "--policy=work-out",
+                 "--add-ingress-zone=HOST"])
+            run(["firewall-offline-cmd",
+                 "--policy=work-out",
+                 "--add-egress-zone=work"])
+            run(["firewall-offline-cmd",
+                 "--policy=work-out",
+                 "--add-service=ssh",
+                 "--add-service=http",
+                 "--add-service=https"])
+            run(["firewall-offline-cmd",
+                 "--policy=work-out",
+                 "--set-target=REJECT"])
+
+            run(["firewall-offline-cmd",
+                 "--new-policy=public-in"])
+            run(["firewall-offline-cmd",
+                 "--policy=public-in",
+                 "--add-ingress-zone=public"])
+            run(["firewall-offline-cmd",
+                 "--policy=public-in",
+                 "--add-egress-zone=HOST"])
+            run(["firewall-offline-cmd",
+                 "--policy=public-in",
+                 "--add-service=ssh",
+                 "--add-service=wireguard"])
+            run(["firewall-offline-cmd",
+                 "--policy=public-in",
+                 "--set-target=REJECT"])
+
+            run(["firewall-offline-cmd",
+                 "--new-policy=public-out"])
+            run(["firewall-offline-cmd",
+                 "--policy=public-out",
+                 "--add-ingress-zone=HOST"])
+            run(["firewall-offline-cmd",
+                 "--policy=public-out",
+                 "--add-egress-zone=public"])
+            run(["firewall-offline-cmd",
+                 "--policy=public-out",
+                 "--add-service=dhcp",
+                 "--add-service=dhcpv6",
+                 "--add-service=http",
+                 "--add-service=https",
+                 "--add-service=wireguard"])
+            run(["firewall-offline-cmd",
+                 "--policy=public-out",
+                 "--set-target=REJECT"])
+
+            print("Firewall configuration complete (will take effect on reboot).")
+            # run(["firewall-cmd", "--reload"])
+
+    def verify(self, args: argparse.Namespace) -> bool:
+        return True
```

Comment:
Is there nothing that can be done here? Is it not worth doing? Is it just not implemented yet? Please put a comment here explaining why this function is empty.

Reply:
It's what would be needed to cause the firewall configuration to apply immediately. I wasn't sure if that's what we wanted, but it looks like quite a lot else applies immediately, and it works, so I'm just going to uncomment this and delete the print.

Reply:
My comment was regarding the `verify` function.